CITATION.md

```diff
@@ -0,0 +1,15 @@
+cff-version: 1.0.0
+title: TUSK
+abstract: Abstract
+authors:
+  - family-names: Druskat
+    given-names: Stephan
+    orcid: "https://orcid.org/0000-0003-4925-7248"
+version: 1.0.0
+date-released: "2021-08-29"
+identifiers:
+  - description: First release
+    type: doi
+    value: "https://zenodo.org/badge/latestdoi/401146593"
```

Comment:
Is there a way we can see the zenodo badge on the repo?

Reply:
We can see zenodo doi in the webhooks section. But for that I have to do the same  process in Neelkanth's repo. Will update you about it.

Reply:
Now you'll be able to see zenodo link in the webhook section of this repo.